packages/fether-react/src/Send/Unlock/Unlock.js

```diff
@@ -90,9 +90,7 @@ class Unlock extends Component {
                     render={({ handleSubmit, pristine, submitting }) => (
                       <form onSubmit={handleSubmit}>
                         <div className='text'>
-                          <p>
-                            Enter your password to confirm this transaction.
-                          </p>
+                          <p>Unlock your account for this transaction:</p>
```

Comment:
I think this could just say 'Unlock account' (the input field already has a button 'Confirm Transaction' below it that makes it clear that it's associated with a transaction)